packages/grid/src/errors/assertIsDefined.ts

```diff
@@ -0,0 +1,11 @@
+import AssertionError from './AssertionError';
+
+export function assertIsDefined<T>(val: T): asserts val is NonNullable<T> {
+  if (val === undefined || val === null) {
+    throw new AssertionError(
```

Comment:
This is also in `assertions.ts`. Guessing you wanted to delete `assertions.ts` and keep this + `AssertionError` files?